ui/tests/acceptance/allocation-detail-test.js

```diff
@@ -240,6 +240,7 @@ module('Acceptance | allocation detail', function(hooks) {
 
       assert.equal(renderedService.name, serverService.name);
       assert.equal(renderedService.port, serverService.portLabel);
+      assert.equal(renderedService.onUpdate, serverService.onUpdate);
```

Comment:
This `onUpdate` property is associated with the above `data-test` attribute you added, but it unfortunately doesn't happen for free.

You'll need to also add this `onUpdate` property [to the page object](https://github.com/hashicorp/nomad/blob/main/ui/tests/pages/allocations/detail.js#L92-L98). That should also make the tests pass.

Reply:
I aborted my review as I was just saying the same thing, otherwise 👍 nice! 😀